packages/http-cors/index.js

```diff
@@ -6,12 +6,19 @@ const defaults = {
 
 const getOrigin = (options, handler) => {
   handler.event.headers = handler.event.headers || {}
-
-  const origin = handler.event.headers['origin'] || handler.event.headers['Origin']
-  if (options.credentials && options.origin === '*' && origin) {
-    return origin
+  const incomingOrigin = handler.event.headers['origin'] || handler.event.headers['Origin']
```

Comment:
I believe we can omit `handler.event.headers['Origin']` as all headers are lower-cased?

Reply:
The original code on this branch checks both?

Reply:
I think we can change the tests to use lowercase only, is that fine @lmammino ?